LambdaEngine/Source/Resources/AnimationGraph.cpp

```diff
@@ -496,6 +543,33 @@ namespace LambdaEngine
 		return false;
 	}
 
+	uint32 AnimationGraph::GetStateIndex(const String& name) const
+	{
+		VALIDATE(m_States.IsEmpty() == false);
```

Comment:
Subjective, do what you feel like.

```suggestion
		VALIDATE(!m_States.IsEmpty());
```